docs/tutorials/kafka-migration.rst

```diff
@@ -0,0 +1,371 @@
+
+* Available **Apache Kafka Connect connectors**: if you're planning to migrate a Kafka Connect cluster, review the list of :doc:`supported connectors </docs/products/kafka/kafka-connect>` or check out how to :doc:`bring your own Kafka Connect cluster </docs/products/kafka/kafka-connect/howto/bring-your-own-kafka-connect-cluster>`. 
+
+* **Connectivity**: the source and target clusters need to be connected via MirrorMaker 2 for the data migration, therefore a proper connection needs to be established. Moreover, you need to ensure all the clients reading or writing to Apache Kafka can connect to the brokers as well as any self hosted Kafka connect cluster. 
+
+  .. Warning::
+
+    As part of the connectivity check you also need to ensure:
+        
+    * That the **connection bandwidth** is high enough to sync the data between environments.
+    * That the needed networking setups are available in the target Apache Kafka cluster, some examples on how to define custom networking setups on Aiven:
+        * :doc:`Firewalls/security groups </docs/platform/concepts/cloud-security>`.
+        * :doc:`VPC peering </docs/platform/howto/manage-vpc-peering>`.
+        * :doc:`Privatelink </docs/platform/howto/use-aws-privatelinks>`.
+
+
+Create an Apache Kafka integration endpoint
+-------------------------------------------
+
+The first step you'll need to perform in the Apache Kafka migration is identifying the source Kafka cluster and create a connection to it. In Aiven:
+
+* Go to the `Aiven Console <https://console.aiven.io/>`_.
+* Click on **Integration Endpoints**.
+  
+  .. image:: /images/tutorials/kafka-migration/integration-endpoints.png
+    :alt: The Aiven Console with the integration endpoints option highlighted
+
+* Select **External Apache Kafka** and click on **Create New**.
+
+  .. image:: /images/tutorials/kafka-migration/external-kafka.png
+    :alt: The Aiven Console with the option to create an External Apache Kafka integration highlighted
+
+* Give the endpoint a **name** (like ``mySourceKafkaCluster``), this will later be used as reference by MirrorMaker 2. Moreover list the **bootstrap servers** and the **Security protocol** that need to be used to connect.
+
+  .. image:: /images/tutorials/kafka-migration/external-kafka-details.png
+    :alt: The list of parameters (endpoint name, bootstrap servers, security protocol) needed to define an External Apache Kafka integration
+
+.. Warning::
+
+    The external integration setup does **not** test the connectivity between Aiven and the source Apache Kafka cluster.
+
+    Always check if additional firewall rules need to be set up to allow MirrorMaker 2 to access your source Apache Kafka cluster.
+
+    Aiven allows you to associate :doc:`static IP addresses </docs/platform/concepts/static-ips>` to all the services, please `contact us <mailto:support@aiven.io>`_ if you need additional help.
```

Comment:
I'm not sure we should include this link to support here.

Reply:
Removed and rephrased, thanks!